indico/modules/rb/controllers/backend/bookings.py

```diff
@@ -5,6 +5,7 @@
 # modify it under the terms of the MIT License; see the
 # LICENSE file for more details.
 
+import typing
```

Comment:
```suggestion
import typing as t
```

and use `t.whatever` accordingly